conf/routes

```diff
@@ -11,17 +11,23 @@ GET         /authenticate/:provider          @controllers.SocialAuthController.a
 POST        /authenticate/credentials        @controllers.CredentialsAuthController.authenticate
 POST        /signUp                          @controllers.SignUpController.signUp
 
+# Group related
 GET         /newGroup                        @controllers.GroupController.newGroupForm
 POST        /newGroup                        @controllers.GroupController.createGroup
 GET         /group/:groupId                  @controllers.GroupController.listGroupPosts(groupId: Long)
 POST        /group/:groupId                  @controllers.PostController.createPost(groupId: Long)
 GET         /editGroup/:groupId              @controllers.GroupController.editGroupForm(groupId: Long)
 POST        /editGroup/:groupId              @controllers.GroupController.updateGroup(groupId: Long)
-GET         /editPost/:postId                @controllers.PostController.editPostForm(postId: Long)
-POST        /editPost/:postId                @controllers.PostController.updatePost(postId: Long)
 GET         /subscribe/:groupId              @controllers.GroupController.subscribe(groupId: Long)
 GET         /unsubscribe/:groupId            @controllers.GroupController.unsubscribe(groupId: Long)
 
+# Post related
+GET         /editPost/:postId                @controllers.PostController.editPostForm(postId: Long)
+POST        /editPost/:postId                @controllers.PostController.updatePost(postId: Long)
+
+# Search
+POST        /search                          @controllers.SearchController.searchPosts
```

Comment:
`/search` is very generic but this only search for posts? So maybe `/posts/search` or something would be better? Or will that search become global search in the future?